client/signup/steps/plans/index.jsx

```diff
@@ -163,12 +165,13 @@ export class PlansStep extends Component {
 					customerType={ this.getCustomerType() }
 					disableBloggerPlanWithNonBlogDomain={ disableBloggerPlanWithNonBlogDomain }
 					plansWithScroll={ true }
+					plans={ plans }
 				/>
 				{ /* The `hideFreePlan` means that we want to hide the Free Plan Info Column.
 				 * In most cases, we want to show the 'Start with Free' PlansSkipButton instead --
 				 * unless we've already selected an option that implies a paid plan.
 				 * This is in particular true for domain names. */
-				hideFreePlan && ! isDomainOnly && ! this.getDomainName() && (
+				hideFreePlan && allowSkip && ! isDomainOnly && ! this.getDomainName() && (
```

Comment:
This line is a bit of a mess. Some times we want to hide the skip button and that should be passed in as a prop rather than adding an endless string of conditional logic. Will need to check all instances of this component and add `allowSkip` where needed.

Reply:
I'm working on a separate PR--I believe the `PlansSkipButton` can be removed all together since we only display the free plan as a separate banner, now.

@jonathansadowski pointed out that this is likely to cause regressions because it's a new prop that won't be set in most cases. In the short term defaulting it to `true` should prevent changes to other flows.